practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/validation/Validations.java

```diff
@@ -4,7 +4,7 @@
  * String constants
  */
 public enum Validations {
-    STARTUP("Laoded practitioner validation with configuration: {}"),
+    STARTUP("Loaded practitioner validation with configuration from config server: {}"),
```

Comment:
I don't think the service should know where it get its config from, this is the whole idea of spring config as it hides the actual sources.